lib/Utils/ConversionUtils/ConversionUtils.cpp

```diff
@@ -193,7 +193,7 @@ struct ConvertFromElements
         newShape.append(shape.begin(), shape.end());
 
         // Create a dense constant for targetShape
-        auto shapeOp = rewriter.create<arith::ConstantOp>(
+        auto shapeOp = rewriter.create<mlir::arith::ConstantOp>(
```

Comment:
Using this instance but there's of course a lot of similar ones: I'm not sure we need to have the `heir::arith` namespace (aka `::mlir::heir::arith`). We already have other passes live in `::mlir::<upstream-dialect>` iirc, so I think we could just put the transform into `::mlir::arith` and avoid a lot of potential for confusion.

Reply:
It is because the ArithToModArith pass lives in the heir::arith namespace.
It is more to be in the same line as the rest of the conversion passes X-to-Y, they all are in namespace X. 
Do you maybe recall with pass it is that lives in an upstream dialect? I will try to migrate it then.

Reply:
I thought the LinAlg-to-Y pass was in `::mlir::linalg` but that's in `::mlir::heir::linalg`, following the same pattern.
I'm not sure if that's a great pattern to follow though, because it does introduce pretty annoying aliasing issue. @j2kun any thoughts on this?

Reply:
I don't see it as particularly annoying, though I would be open to having more unique names. I don't think it should block this PR